PyQt5-stubs/QtCore.pyi

```diff
@@ -40,9 +40,22 @@ class pyqtBoundSignal:
     @typing.overload
     def disconnect(self, slot: typing.Union["PYQT_SLOT", "QMetaObject.Connection"]) -> None: ...
 
-
+_SignalTypesT = typing.TypeVar("_SignalTypesT", type, typing.List[type])
```

Comment:
```suggestion
_SignalTypesT = typing.TypeVar("_SignalTypesT", typing.Union[type, typing.List[type]])
```
I can't say I know the reasons for why this works and the separate types don't but...  it does.  `:|`  Is there something that makes it wrong despite passing?

Reply:
Err, hold that thought...  maybe I didn't test properly here...

Reply:
Ok, trying again and hoping to be less silly.  As is, there is no relationship being created between multiple hints so this can just be an alias instead, I think?

```suggestion
_SignalTypesT = typing.Union[type, typing.List[type]]
```
(except with a different variable name changed here and at the use sites)

Reply:
The issue here is that I need to keep all of the var args of the same type. Using the Union allows for
```python
a = pyqtSignal([int, bool], float)
```
but this is not allowed by PyQt. You can use _only_ types OR _only_ lists of types

Reply:
Well, I think my second attempt was _less_ silly...  though still not helpful.  Humm.